layout/_macro/post.swig

```diff
@@ -1,6 +1,6 @@
 {% macro render(post, is_index, post_extra_class) %}
 
-  {% set headlessPost = Array.prototype.indexOf.call(['quote', 'picture'], post.type) > -1 %}
+  {% set headlessPost = Array.prototype.indexOf.call(['quote', 'picture', 'nohead'], post.type) > -1 %}
```

Comment:
`header: false` option added in https://github.com/theme-next/hexo-theme-next/commit/92fdf867bf8d1ae146d318cfd5f134c26bae7671#diff-0e030eb6dd7b9d25bfcba06bc8ea5609R27